### DISCUSSION TITLE
Update auf neue Version

### DISCUSSION THREAD

**Discussion by Deanw1975** [solectrus/solectrus#2162] [Q&A]
Servus, 

ich habe mit viel mühe und lesen solectrus installiert.

Nun wollte ich auf die neue Version updaten.

Mit dem Befehl "docker compose pull" bekomme ich diese Fehlermeldung:

"no configuration file provided: not found"

Was kann ich tun?

VG
Dean

**Comment by ledermann** [comment]
Das Update ist eigentlich ganz einfach, also wesentlich einfacher als die Erstinstallation :-) 

Die Fehlermeldung deutet darauf hin, dass Du nicht im richtigen Ordner bist. Du musst mit `cd` vorher in den Ordner wechseln, in dem die `docker-compose.yml` liegt. Dann sollte es klappen.

**Comment by Deanw1975** [comment]
OK das wars also ;-)
Danke für die schnelle Hilfe. Mal schauen ob ich beim nächsten Update an den Ordnerwechsel denke ;-)

VG
Dean
